packages/gibsonify_api/lib/src/models/recipe_ingredient.dart

```diff
@@ -1,278 +1,77 @@
-
-  @override
-  DescriptionValidationError? validator(String? value) {
-    return value?.isNotEmpty == true
-        ? null
-        : DescriptionValidationError.invalid;
-  }
-}
-
-enum CookingStateValidationError { invalid }
-
-class CookingState extends FormzInput<String, CookingStateValidationError> {
-  const CookingState.pure() : super.pure('');
-  const CookingState.dirty([String value = '']) : super.dirty(value);
-
-  Map<String, dynamic> toJson() {
-    final Map<String, dynamic> data = <String, dynamic>{};
-    data['value'] = value;
-    data['pure'] = pure.toString();
-    return data;
-  }
-
-  CookingState.fromJson(Map<String, dynamic> json) : super.dirty(json['value']);
-
-  @override
-  CookingStateValidationError? validator(String? value) {
-    return value?.isNotEmpty == true
-        ? null
-        : CookingStateValidationError.invalid;
-  }
-}
-
-enum MeasurementMethodValidationError { invalid }
-
-class MeasurementMethod
-    extends FormzInput<String, MeasurementMethodValidationError> {
-  const MeasurementMethod.pure() : super.pure('');
-  const MeasurementMethod.dirty([String value = '']) : super.dirty(value);
-
-  Map<String, dynamic> toJson() {
-    final Map<String, dynamic> data = <String, dynamic>{};
-    data['value'] = value;
-    data['pure'] = pure.toString();
-    return data;
-  }
-
-  MeasurementMethod.fromJson(Map<String, dynamic> json)
-      : super.dirty(json['value']);
-
-  @override
-  MeasurementMethodValidationError? validator(String? value) {
-    return value?.isNotEmpty == true
-        ? null
-        : MeasurementMethodValidationError.invalid;
-  }
-}
-
-enum MeasurementValidationError { invalid }
+  List<Object?> get props =>
+      [name, otherName, description, cookingState, measurements, saved, id];
 
-class Measurement extends FormzInput<String, MeasurementValidationError> {
-  const Measurement.pure() : super.pure('');
-  const Measurement.dirty([String value = '']) : super.dirty(value);
-
-  Map<String, dynamic> toJson() {
-    final Map<String, dynamic> data = <String, dynamic>{};
-    data['value'] = value;
-    data['pure'] = pure.toString();
-    return data;
-  }
-
-  Measurement.fromJson(Map<String, dynamic> json) : super.dirty(json['value']);
-
-  @override
-  MeasurementValidationError? validator(String? value) {
-    return value?.isNotEmpty == true
-        ? null
-        : MeasurementValidationError.invalid;
-  }
-}
-
-enum MeasurementUnitValidationError { invalid }
-
-class MeasurementUnit
-    extends FormzInput<String, MeasurementUnitValidationError> {
-  const MeasurementUnit.pure() : super.pure('');
-  const MeasurementUnit.dirty([String value = '']) : super.dirty(value);
-
-  Map<String, dynamic> toJson() {
-    final Map<String, dynamic> data = <String, dynamic>{};
-    data['value'] = value;
-    data['pure'] = pure.toString();
-    return data;
+  static Future<String> getIngredients() {
+    return rootBundle.loadString('assets/ingredients/ingredients.json');
```

Comment:
Hmm, I'm not sure if this should be a part of the recipe_ingredient model in the API or a part of the BLoC. But I guess it can be left like this for now.